auth/user_mgt_test.go

```diff
@@ -875,9 +909,11 @@ func TestInvalidUpdateUser(t *testing.T) {
 			(&UserToUpdate{}).MFASettings(MultiFactorSettings{
 				EnrolledFactors: []*MultiFactorInfo{
 					{
-						UID:         "enrolledSecondFactor1",
-						PhoneNumber: "+11234567890",
```

Comment:
Can we also add a test case for the deprecated PhoneNumber for UpdateUser, please?